tests/validator.go

```diff
@@ -147,7 +147,11 @@ func validateFilesystems(t *testing.T, expected []*types.Partition) error {
 			e.FilesystemLabel == "" {
 			continue
 		}
-		info, err := util.GetFilesystemInfo(e.Device)
+		strict := true
+		if e.Ambivalent {
+			strict = false
+		}
+		info, err := util.GetFilesystemInfo(e.Device, strict)
```

Comment:
I think it'd be fine if you wanted to drop this to `util.GetFilesystemInfo(e.Device, !e.Ambivalent)` but leaving it as is would be fine too.